auto_nag/scripts/configs/dom_core_round_robin.json

```diff
@@ -0,0 +1,25 @@
+{
+    "fallback": "Hsin-Yi Tsai",
+    "components": {
+        "Core::DOM: Core & HTML": "default",
+        "Core::DOM: Editor": "default",
+        "Core::DOM: HTML Parser": "default",
+        "Core::DOM: Serializers": "default",
+        "Core::XML": "default",
+        "Core::XPConnect": "default",
+        "Core::DOM: Selection": "default",
+        "Core::DOM: Navigation": "default",
+        "Core::XSLT": "default",
+        "Core::DOM: Copy & Paste and Drag & Drop": "default",
+        "Core::DOM: Events": "default",
+        "Core::DOM: Geolocation": "default",
+        "Core::DOM: Bindings (WebIDL)": "default",
+        "Core::DOM: UI Events & Focus Handling": "default",
+        "Core::DOM: Forms": "default",
+        "Core::DOM: Streams": "default",
```

Comment:
This component should be excluded even though it is in https://bugzilla.mozilla.org/rest/config/component_teams/DOM%20Core because the triage owner is in the JavaScript team.